server.go

```diff
@@ -956,10 +970,24 @@ func (s *server) Start() error {
 		return nil
 	}
 
+	// If one sub system fails to start, the following code
+	// ensures that the  previous started ones are stopped.
```

Comment:
Nit: Extend comments to 80 chars per line. Also there's an extra space after `the`.

Reply:
fixed